packages/stratocacher-layer-dynamo/src/index.js

```diff
@@ -29,6 +30,13 @@ function emitError(message) {
 	v       String
 	t       Number
 	i       Boolean
+	c       Boolean
```

Comment:
Would be nice to have mnemonics for what these single-character names _mean_ in addition to what type they are here.